glean-core/ios/Glean/Utils/Sysctl.swift

```diff
@@ -0,0 +1,158 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import Foundation
+
+// swiftlint:disable force_try
+// REASON: Used on infallible operations
+
+// swiftlint:disable identifier_name
+// REASON: short names are ok
+
+// swiftlint:disable line_length
+// REASON: URLs and doc strings
+
+/// Original: https://github.com/mattgallagher/CwlUtils/blob/0e08b0194bf95861e5aac27e8857a972983315d7/Sources/CwlUtils/CwlSysctl.swift
+/// Copyright © 2016 Matt Gallagher ( https://www.cocoawithlove.com ). All rights reserved.
+/// License: ISC (https://github.com/mattgallagher/CwlUtils/blob/0e08b0194bf95861e5aac27e8857a972983315d7/LICENSE.txt)
```

Comment:
Mh. can we even do that? Are licenses compatible? Should we hear legal?

Reply:
Should be, I double-check

Reply:
We're fine: ![](https://tmp.fnordig.de/scr/178ccb05dd.png)

> The following licences are compatible with the MPL, and code under these licenses can be integrated into Mozilla products.
> [...]
> The ISC license

(from our Licensing Runbook, also https://www.gnu.org/licenses/license-list.en.html#ISC)